code/game/machinery/chemHandC.dm

```diff
@@ -0,0 +1,264 @@
+
+//Heater
+
+/obj/machinery/chemheater
+	name = "directed laser heater"
+	desc = "A small stage where a container can be placed to heat it's contents with a laser."
```

Comment:
it's vs its
You can probably rephrase the whole thing to be better

Reply:
I can work on it but I'm open to suggestions from other people.

Reply:
A platform with an integrated laser that uses high-energy photons to heat a subject through atomic vibrations. In a practical sense, it has no upper limit to how much thermal energy can be induced this way, as it is capable of reaching temperatures which will rapidly destroy any laboratory-approved containers.

I for one welcome our melting point overlords.

Reply:
I will play around with this likely and mirror it to the cooler unless you got other guidelines for the cooler in mind.